packages/frontend-api/src/Resources/config/graphql-types/PriceInterfaceDecorator.types.yaml

```diff
@@ -0,0 +1,15 @@
+PriceInterfaceDecorator:
```

Comment:
Why do we need this? I see only one usage of the interface

Reply:
removed in aa6097c367